### ISSUE TITLE
logistic regression: loss function

### ISSUE THREAD

**Issue by Sarnath** [aymericdamien/TensorFlow-Examples#86]
The logistic loss in basic_lesson2 exercise is shown as

`cost = tf.reduce_mean(-tf.reduce_sum(y*tf.log(pred), reduction_indices=1))`

While this considers the loss of label 1 misclassified as 0, this does not seem to consider the loss of label 0 misclassified as 1 which is `(1-y)*log(1-pred)`
